pkg/accounting/main_test.go

```diff
@@ -0,0 +1,13 @@
+// Copyright 2022 The Swarm Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package accounting_test
```

Comment:
if these leak tests fall under the _test file, why not add them to the already existing test file?

Reply:
Only function defined in `main_test.go` files is `TestMain` function which does not do any testing per say. 

`TestMain` is common function for all tests which in this case adds logic for checking goroutine leaks in all `_test` files of current package. Since it adds additional logic to all tests it's makes more sense to be in individual file because in case when package has multiple `_test` files it wouldn't be clear which file should host common logic.

Reply:
@istae ping